### DISCUSSION TITLE
why prisma package is nearly 100 mb ?

### DISCUSSION THREAD

**Discussion by HosMercury** [prisma/prisma#8269] [Q&A] [answered]
why prisma package is nearly 100 mb ? very huge

**Comment by ryands17** [comment]
@HosMercury 👋 

How are you packaging Prisma currently?

**Comment by HosMercury** [comment, marked as answer]
Express app then npm install prisma .

**Reply by ryands17** [reply]
That's because Prisma comes with the necessary engines for you to run the queries, migration, introspection, etc. For development that's required, but once you deploy to production, just the `query-engine` is needed present in the `node_modules/.prisma` folder so folders like `node_modules/prisma` and `node_modules/@prisma/engines` is not needed and can be deleted.

This will make your production size much smaller.

**Reply by HosMercury** [reply]
Ok thanks for clarification

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
